src/AutoRest.CSharp/LowLevel/Generation/LowLevelClientWriter.cs

```diff
@@ -93,8 +92,8 @@ public void WriteClient()
                         var longRunning = clientMethod.LongRunning;
                         if (longRunning != null)
                         {
-                            WriteLongRunningOperationMethod(clientMethod, client.Fields, longRunning, true);
-                            WriteLongRunningOperationMethod(clientMethod, client.Fields, longRunning, false);
+                            WriteLongRunningOperationMethods(clientMethod, client.Fields, longRunning, true);
```

Comment:
I prefer not to changing the method name.

Reply:
See https://github.com/Azure/autorest.csharp/issues/2552 item 3 `Consolidate all methods created for one operation`.